### PR TITLE
OSDOCS-7860: Migrate "Federating System and User Metrics to S3 in ROSA" to ROSA product documentation

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-7860](https://issues.redhat.com//browse/OSDOCS-7860): Migrate "Federating System and User Metrics to S3 in ROSA" to ROSA product documentation

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-7860

Link to docs preview:
https://65047--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/rosa-mobb-federating-system-user-metrics-s3-rosa

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: rosa_tutorials/rosa-mobb-federating-system-user-metrics-s3-rosa.adoc [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+.. Ensure that the user workload configuration map exists:
++
+[source,terminal]
+----
+$    oc -n openshift-user-workload-monitoring get \
+      configmaps user-workload-monitoring-config
+----
+
+.. If the config doesn't exist run:
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-federating-system-user-metrics-s3-rosa.adoc:207]:
```suggestion
.. If the config does not exist run:
```

File: rosa_tutorials/rosa-mobb-federating-system-user-metrics-s3-rosa.adoc [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+          ...
+          prometheus:
+          ...
+            remoteWrite:
+              - url: "http://thanos-receive.thanos-receiver.svc.cluster.local:9091/api/v1/receive"
+----
+
+== Ensure that metrics are flowing by logging into Grafana
+
+. Obtain the route URL for Grafana (remember it's https) and log in using username `root` and the updated password or the default of `secret`.
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-federating-system-user-metrics-s3-rosa.adoc:242]:
```suggestion
. Obtain the route URL for Grafana (remember it is https) and log in using username `root` and the updated password or the default of `secret`.
```

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
@jneczypor I only suggested the contractions because of possible translation.

**Comment by jneczypor** [comment]
Hi @michaelryanmcneill - Can you please review this PR? Thanks so much!

**Comment by michaelryanmcneill** [comment]
@jneczypor it appears that the code blocks on this article are broken, can you fix the indentation and circle back with me so I can review?

**Pull request closed without merging**